lib/optimizely/odp/odp_config.rb

```diff
@@ -30,6 +32,7 @@ def initialize(api_key = nil, api_host = nil, segments_to_check = [])
       @api_host = api_host
       @segments_to_check = segments_to_check
       @mutex = Mutex.new
+      @odp_state = @api_host.nil? || @api_key.nil? ? ODP_CONFIG_STATE[:UNDETERMINED] : ODP_CONFIG_STATE[:INTEGRATED]
```

Comment:
I don't have a strong opinion either way, but apparently Ruby does because this is explicitly called out in the style guide:
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/TernaryParentheses 😄

Reply:
Ahhh!! Rubocop is brutal! I take my suggestion back :)